File: bisc_2018/o.py
import math

a = input ()
a = int (a)

arr = list (range (1, a + 1))

lcm = 0
for i in range (1, a + 1):
    if i == 1:
        lcm = i
    else:
        lcm = (lcm * i) // math.gcd (i, lcm)

print (lcm)

File: bisc_2018/o.cpp
#include <bits/stdc++.h>

void input ();
void solve ();

const int MAXN = 1e3 + 1;

typedef long long ll;

int n;
ll arr [MAXN];

void input () {
	scanf ("%d", &n);
	solve ();
}

void solve () {
	for (ll i = 1; i <= n; i++)
		arr [i - 1] = i;

	ll lcm = (ll) arr [0];
	for (int i = 1; i < n; i++)
		lcm = (arr [i] * lcm) / std::__gcd (std::min (arr [i], lcm), std::max (arr [i], lcm));
	printf ("%lld\n", lcm);
}

int main () {
	input ();
	return 0;
}
